app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -115,7 +116,7 @@ class BrowserFragment : Fragment(), BackHandler, CoroutineScope,
         savedInstanceState: Bundle?
     ): View? {
         require(arguments != null)
-        customTabSessionId = BrowserFragmentArgs.fromBundle(arguments!!).customTabSessionId
+        customTabSessionId = arguments?.getString(IntentProcessor.ACTIVE_SESSION_ID)
```

Comment:
I'm not sure what the purpose of this change is. Before, it was using Safe Args as compile-time validation of the argument name and nullability. Now, it's not.

Reply:
The `BrowserFragmentArgs` name disagreed with how the `IntentProcessor` was putting the session id into the bundle: i.e. the key derived from the [navigation graph][1]: `custom_tab_session_id` != the [key from the IntentProcessor][2]: `activeSessionId`

Given 1) we have access to the _actual_ key directly, and 2) the `getString()` returns `null` if it's not there, I opted against using the _copy of_ the key from`BrowserFragmentArgs` — it's not really providing us with any type safety if we're not using the generated data class from the sending side.

Happy use `SafeBundle` to check against a `ClassCastException` or some other `RuntimeException`.

Please advise.

[1]: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/navigation/nav_graph.xml#L107
[2]: https://github.com/mozilla-mobile/android-components/blob/51ef74db39a20fc76d62775db0f2039a6d0959bd/components/feature/intent/src/main/java/mozilla/components/feature/intent/IntentProcessor.kt#L142

Reply:
I suppose it's okay. It just seems odd, given that we're only passing in the value by Safe Args. I mean I spent most of my career without any Safe Args, but it seems silly to add them to the nav graph and then not use them. I'd either get the nullable value from Safe Args, like we were, or remove it from the graph as unused.